_site/404.html

```diff
@@ -1,126 +1,109 @@
-<!DOCTYPE html>
-<html lang="en"><head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- Begin Jekyll SEO tag v2.8.0 -->
 <title>Elijah Belnap | Elijah Belnap is a Data Engineer enthusiastic about various data technologies. On his website he shares his knowledge and experience regarding SQL Server, Postgres, MySQL and more.</title>
 <meta name="generator" content="Jekyll v4.3.3" />
 <meta property="og:title" content="Elijah Belnap" />
 <meta property="og:locale" content="en_US" />
 <meta name="description" content="Elijah Belnap is a Data Engineer enthusiastic about various data technologies. On his website he shares his knowledge and experience regarding SQL Server, Postgres, MySQL and more." />
 <meta property="og:description" content="Elijah Belnap is a Data Engineer enthusiastic about various data technologies. On his website he shares his knowledge and experience regarding SQL Server, Postgres, MySQL and more." />
-<link rel="canonical" href="https://elijahbelnap.com/404.html" />
-<meta property="og:url" content="https://elijahbelnap.com/404.html" />
+<link rel="canonical" href="http://localhost:4000/" />
```

Comment:
I noticed that we're replacing all the links that used to use the website url with these local host links, which I'm curious as to how that will affect the website when uploaded to the webserver. Also, we switched from serving over https to serving through http.

Reply:
I didn't change those manually, it must've been through Jekyll but I'm unsure why it would do that.